server/backend/migration/migration.go

```diff
@@ -71,6 +71,7 @@ func manualMigrate(db *gorm.DB) error {
 		migrate20240102000000(),
 		migrate20240103000000(),
 		migrate20240207000000(),
+		migrate20240311000000(),
 		migrate20240212000000(),
```

Comment:
> why did you ad the migration with an earlier timestamp instead of the current date

That was a typo `migrate20240212000000` should have been `migrate20240312000000`.
TBH, I am going to reoder this in the list and call it a day.
Said typo is not quite worth fixing imo..

```suggestion
		migrate20240212000000(),
		migrate20240311000000(),
```